demo_endtoend_imagenet.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from scivision_test_plugin import ImageNetModel
</pre><p>If you make the other changes, you should be able to remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/scivision/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>